tap/extensions/amqp/main.go

```diff
@@ -78,15 +78,17 @@ func (d dissecting) Dissect(b *bufio.Reader, isClient bool, tcpID *api.TcpID, co
 	var lastMethodFrameMessage Message
 
 	for {
+		// fmt.Printf("lastMethodFrameMessage: %v\n", lastMethodFrameMessage)
```

Comment:
Either remove comment or print in DEBUG or TRACE level.
Same comment for line 90.

Reply:
@nimrod-up9 removed with https://github.com/up9inc/mizu/commit/1a6d32e6614e4587883b27dda599f011054b8605